SparkPost/2web2ui
# SA-1324 Allow from and to params in health score dashboard page

### What Changed
 - If to and from are given in the health score dashboard URL, update redux with new from/to and update the url to remove the from and to params.

### How To Test
 - Go to /signals/health-score?from={date}&to={date}
 - Verify that the correct dates are used.

### To Do
- [ ] Possible always have to and from params showing?
- [ ] Can't seem to stop the table from fetching data twice.?
